mods/cubyz/rotation/stairs.zig

```diff
@@ -276,9 +279,12 @@ fn closestRay(comptime typ: enum {bit, intersection}, block: Block, relativePlay
 pub fn rayIntersection(block: Block, item: ?main.items.Item, relativePlayerPos: Vec3f, playerDir: Vec3f) ?RayIntersectionResult {
 	if(item) |_item| {
 		switch(_item) {
-			.baseItem => |baseItem| {
-				if(std.mem.eql(u8, baseItem.id(), "cubyz:chisel")) { // Select only one eighth of a block
-					return closestRay(.intersection, block, relativePlayerPos, playerDir);
+			.tool => |tool| {
+				const tags = tool.type.blockTags();
+				for(tags) |tag| {
+					if(std.mem.eql(u8, tag.getName(), blockTag)) {
```

Comment:
Never compare tags by name, the whole point of using an enum is to allow comparing by enum value.